go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/codeocean/go-mlflow
+
+go 1.19
```

Comment:
Is there a reason not to use the latest 1.22 release?

Reply:
This would enforce everyone on the team to move forward to 1.22 since the `gw` will need to move forward as well as it'll depend on this.